### PR TITLE
Update sleep.md

### PR DESCRIPTION
**Pull request by eyadsalama**
@casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2153)

### CODE REVIEW

File: arabic/sleep/sleep.md [hunk 1]
```diff
@@ -3,3 +3,6 @@
 لذا ماذا ستفعل الآن؟
 نعم كلامك صحيح  نحن في حاجة إلى العلم ولكن من العوامل الأساسية التي تساعد على تحصيل العلم جيداً هي أخذ قسط
 كافي من النوم لذا سوف أختار أن أنام الآن .  
+كانت رحلة جميلة قضيتها مع زوجتى فى احدى المنتجعات الرائعة وبينما نحن فى طريقنا للعودة ليلا : اذ يالعربة تتعطل فجأة
+ماذا ستفعل؟
+1-[حاول تصليح العربة] (../arabic/sleep/try-fix.md)
```
Comment (casey-collab) [inline arabic/sleep/sleep.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by eyadsalama** [comment]
@casey-collab

File: arabic/sleep/sleep.md [hunk 2]
```diff
@@ -3,3 +3,6 @@
 لذا ماذا ستفعل الآن؟
 نعم كلامك صحيح  نحن في حاجة إلى العلم ولكن من العوامل الأساسية التي تساعد على تحصيل العلم جيداً هي أخذ قسط
 كافي من النوم لذا سوف أختار أن أنام الآن .  
+كانت رحلة جميلة قضيتها مع زوجتى فى احدى المنتجعات الرائعة وبينما نحن فى طريقنا للعودة ليلا : اذ يالعربة تتعطل فجأة
+ماذا ستفعل؟
+1-[حاول تصليح العربة] (../arabic/fix/try-fix.md)
```
Comment (casey-collab) [inline arabic/sleep/sleep.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: arabic/fix/try-fix.md [hunk 3]
```diff
@@ -0,0 +1 @@
+حاولت قدر المستطاع اصلاع العربة ولكن بلا فائدة
```
Comment (casey-collab) [inline arabic/fix/try-fix.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by eyadsalama** [comment]
@casey-collab

File: arabic/sleep/sleep.md [hunk 4]
```diff
@@ -3,3 +3,6 @@
...
```
Comment (casey-collab) [inline arabic/sleep/sleep.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: arabic/fix/try-fix.md [hunk 5]
```diff
@@ -0,0 +1 @@
+ حاولت قدر المستطاع اصلاع العربة ولكن بلا فائدة
```
Comment (casey-collab) [inline arabic/fix/try-fix.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: arabic/sleep/sleep.md (hunk 4 above)
Comment (cbuckey-uda) [inline arabic/sleep/sleep.md]: @eyadsalama, you're on the right track! I believe the link `../fix/try-fix.md` would work, and that would also fix the problem where the other page is unreachable.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**